integration/tests/basic/helpers/basic.py

```diff
@@ -1,20 +1,6 @@
-import time
-import typing as tp
 from dataclasses import dataclass
 from enum import Enum
 
-import allure
-import eth_account.signers.local
-import pytest
-import web3
-from solders.rpc.responses import GetTransactionResp
-from solders.signature import Signature
-
-from integration.tests.base import BaseTests
-from utils.apiclient import JsonRPCSession
-from utils.consts import Unit, InputTestConstants
-from utils.helpers import gen_hash_of_block, wait_condition
-
 
 @dataclass
 class AccountData:
```

Comment:
looks like we can delete it too
Or move at least

Reply:
it uses in several tests